index.js

```diff
@@ -0,0 +1,12 @@
+var express = require( "express" );
+var app = express();
+
+//jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+var port = process.env.npm_package_config_port;
+
+//jscs:enable requireCamelCaseOrUpperCaseIdentifiers
+
+app.use ( "/", express.static( "./static" ) );
+app.listen( port, function() {
+	console.log( "Listening on http://localhost:" + port );
```

Comment:
Yes, that's fine - its just a convenience when testing locally. If it turns out later I'm wrong about it, we can still fix it quickly enough...